lib/web/integrations_awsoidc.go

```diff
@@ -1454,6 +1454,10 @@ func (h *Handler) awsOIDCPing(w http.ResponseWriter, r *http.Request, p httprout
 		return nil, trace.Wrap(err)
 	}
 
+	if req.RoleARN != "" {
+		integrationName = ""
+	}
+
```

Comment:
this is supposed to remove any integrationName even if that exists? if it is required for roleARN to exist for integration name to exist, should we do validation instead? (!(roleArn == "" && integrationName == ""))

Reply:
the block is basically enforcing, either `roleArn` is defined OR `integrationName` but not both

Reply:
Oh I read that backwards (thought it was if it was missing). Ok thank you, sounds good